CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Coverage Upload Action Changelog
 
+## 3.0.0 (June 20, 2022)
```

Comment:
Is this actually a breaking API change? Is there any way we can make it not be a breaking change?

Reply:
I don't think so, I tried for it not to be a breaking change. But if it's a monorepo we have to get all the test information from all the `lcov.info` files that are configured so that we can make a coverage report comment on that monorepo.

In order to group all that information together and post it to the endpoint we need to change the PR data that we post into an array instead of just an object.
I think that changing the data that the endpoint should expect classifies as a breaking change, unless my understanding is wrong here.